.travis.yml

```diff
@@ -1,6 +1,13 @@
 language: node_js
+cache:
+  directories:
+    - node_modules
+
+notifications:
+  email: false
+
 node_js:
-  - '4'
+  - '6'
```

Comment:
We had been using node 4 because of build issues. We can try it, but might need to fall back here?